templates/events/notification.html

```diff
@@ -69,9 +69,9 @@ <h4 class="modal-title">Confirmation</h4>
                   <div class="form-group row">
                     <label for="confirmNotification" class="col-12 col-form-label"><center>Do you want to send this notification?</center></label>
                   </div>
```

Comment:
The text box in notification popup modal need to match the zeplin design (no rounded border. remove border radius).
Other than that, it looks good to me.
Approving with minor change requested above.

Reply:
sounds good.